src/containers/MapExplorer.jsx

```diff
@@ -57,7 +56,7 @@ class MapExplorer extends Component {
             {this.props.selectors.map((selector) => {
               selector.tmp = selector.id + selector.id;
               return (
-                <SelectorPanel ref={'selectorPanel_' + selector.id} key={selector.id} selectorData={selector} />
+                <MapControls ref={'mapControls_' + selector.id} key={selector.id} selectorData={selector} />
```

Comment:
In that case, there's a lot in containers/MapExplorer and containers/MapControls that can benefit from String interpolation. Gimme a few minutes, I'll see if I can do a mass cleanup.